Autopilot/AttitudeManager/Network/Datalink.h

```diff
@@ -14,16 +14,16 @@
 #include <stdbool.h>
 
 /** Time in miliseconds for how often a P0(high priority) packet gets sent down. Default=300 **/
-#define P0_SEND_FREQUENCY 200 
+#define P0_SEND_FREQUENCY 250 
 
 /** Time in miliseconds for how often a P1(medium priority) packet gets sent down. Default=1000 **/
-#define P1_SEND_FREQUENCY 800
+#define P1_SEND_FREQUENCY 1000
 
 /** Time in miliseconds for how often a P2(low priority) packet gets sent down. Default=20000 **/
 #define P2_SEND_FREQUENCY 5000
 
 /** Time in miliseconds for how often to check for new messages from the uplink. Default=100 **/
-#define UPLINK_CHECK_FREQUENCY 75
+#define UPLINK_CHECK_FREQUENCY 500
```

Comment:
This is sorta like a preventative measure against spamming commands. ie. if you're doing it, I dont want the datalink downlink speed to decrease as the picpilot parses stuff, which it will since 75 is lower than everything else. I'm still playing around with these values to find what I think is best.